cli/src/program.rs

```diff
@@ -2414,6 +2429,15 @@ fn process_extend_program(
     let upgrade_authority_address = upgrade_authority_address
         .ok_or_else(|| format!("Program {program_pubkey} is not upgradeable"))?;
 
+    if authority_signer.pubkey() != upgrade_authority_address {
+        return Err(format!(
+            "Upgrade authority {} does not match {}",
+            upgrade_authority_address,
+            Some(authority_signer.pubkey())
```

Comment:
The clippy job is failing because of this line -- is there a reason to print an `Option(Pubkey)` over just the raw `Pubkey`?